### DISCUSSION TITLE
How do I configure blog on home page but have URLs with subfolders for each year and month?

### DISCUSSION THREAD

**Discussion by pftnhr** [datenstrom/community#881] [Ask a question] [answered]
The background to my question is the fact that I would like to store the `post.md` files in a `year/month/` folder structure, but the folder `/content/1-home` must not have any subfolders.

I am currently working with a redirect in the page settings, which I do not like for various reasons.

I look forward to easy to implement answers. 😄

**Comment by markseuffert** [comment]
You can do this by adjusting your blog settings and content structure.

Have a look at [#239](https://github.com/datenstrom/community/discussions/239#discussioncomment-244744), example 6 should explain the setup your want.

**Reply by pftnhr** [reply]
But my inner Monk wants the folders `year/month` to be in the blog folder. 🙈

**Reply by markseuffert** [reply, marked as answer]
I think you will find that example 3 is a better starting point for the setup you want. You can define the blog start location to be anywhere, it does not have to be in the blog folder. For example configure the blog setting `BlogStartLocation: /` and then edit the file `content/1-home/page.md` as following:

```
---
Title: Home
Layout: blog-start
LayoutNew: blog
---
```

How lucky that we support the Vulcan concept of infinite diversity in infinite combination. :vulcan_salute:

**Reply by pftnhr** [reply]
That's it! Thanks a lot!
